packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -110,6 +111,7 @@ export type GraphiQLState = {
   subscription?: Unsubscribable | null;
   variableToType?: VariableToType;
   operations?: OperationDefinitionNode[];
+  parameters?: any;
```

Comment:
we are not adding state here, the goal is to eliminate state in GraphiQL component. this logic belongs in SessionProvider